src/v2/Components/BankDebitForm/BankDebitProvider.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC, useEffect, useState } from "react"
+import { Appearance, loadStripe } from "@stripe/stripe-js"
+import { Elements } from "@stripe/react-stripe-js"
+import { getENV } from "v2/Utils/getENV"
+import { BankDebitForm } from "./BankDebitForm"
+import { CreateBankDebitSetupForOrder } from "./Mutations/CreateBankDebitSetupForOrder"
+import { Payment_order } from "v2/__generated__/Payment_order.graphql"
+interface Props {
+  order: Payment_order
+}
+
+export const BankDebitProvider: FC<Props> = ({ order }) => {
+  const [clientSecret, setClientSecret] = useState("")
+  const stripePromise = loadStripe(getENV("STRIPE_PUBLISHABLE_KEY"))
+  const { submitMutation } = CreateBankDebitSetupForOrder()
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await submitMutation({
+        variables: { input: { id: order.internalID } },
+      })
+
+      if (
+        data.commerceCreateBankDebitSetupForOrder?.actionOrError.__typename ===
+        "CommerceOrderRequiresAction"
```

Comment:
Created a ticket for error handling here: https://artsyproduct.atlassian.net/browse/TX-285

Reply:
Thanks!